zeq/publisher.h

```diff
@@ -44,6 +45,9 @@ class Publisher
 
     const std::string& getAddress() const; //!< @internal
 
+    /** @return the publisher URI given in the constructor. */
+    ZEQ_API const lunchbox::URI& getURI() const;
```

Comment:
I made the same comment yesterday but I don't find it anywhere. Where did it go...?